src/style.css

```
@@ -0,0 +1,43 @@
+
+html {
+  box-sizing: border-box;
+}
+
+* {
+  margin: initial;
+}
```

```suggestion
* {
   box-sizing: border-box;
}
```
in mostly cases * selector only for box-sizing property. you can change margin directly for elements where it is necessary